envs/example/ci-ceph_swift/group_vars/all.yml

```diff
@@ -40,6 +40,8 @@ cinder:
   enabled_backends: [] # Determined automatically by Ceph roles
   backends: []
     # Determined automatically by Ceph roles
+lvm:
```

Comment:
only 2 fields install qemu-utils, if no need to enable lvm for ceph, we need rewrite condition to install it
./ursula/roles/cinder-common/defaults/main.yml:      - qemu-utils
./ursula/roles/ironic-data/tasks/main.yml:    - qemu-utils